index.html

```diff
@@ -1,55 +1,247 @@
+            </div>
+          </div>
+          <div class="box">
+            <div></div>
+            <div class="box-menu">
+              <h2 class="title">Multi-Post Stories Gain+Glory</h2>
+              <ul class="box-list">
+                <li class="box-list-item">Ruby on rails</li>
+                <li class="box-list-item">css</li>
+                <li class="box-list-item">Javascript</li>
+                <li class="box-list-item">html</li>
+              </ul>
+              <button class="box-button">See Project</button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="third-section">
+      <div class="container">
+        <div class="frameleft">
+          <img  src="./Images/frameleft.svg" alt="">
+        </div>
+        <div class="frameright"> <img  src="./Images/frameright.svg" alt="">
+        </div>
+        <h2>About Me</h2>
+        <p class="about-desc">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <button class="box-button about-btn">Get My Resume</button>
+      </div>
+      <ul class="boxes">
+        <li class="about-box">
+          <div><img class="about-img" src="./Images/icon1.svg" alt="" /></div>
+          <div class="about-menu">
+            <h2 class="about-title">Languages</h2>
+            <ul class="about-list">
+              <li class="about-list-item">Javascript</li>
+              <li class="about-list-item">Ruby</li>
+              <li class="about-list-item">Html</li>
+              <li class="about-list-item">CSS</li>
+            </ul>
+          </div>
+        </li>
+        <li class="about-box">
+          <div><img class="about-img" src="./Images/icon2.svg" alt="" /></div>
+          <div class="about-menu">
+            <h2 class="about-title">Frameworks</h2>
+            <ul class="about-list">
+              <li class="about-list-item">Bootstrap</li>
+              <li class="about-list-item">Ruby on rails</li>
+              <li class="about-list-item">RSpec</li>
+              <li class="about-list-item">Capybara</li>
+              <li class="about-list-item">Selenium</li>
+            </ul>
+          </div>
+        </li>
+        <li class="about-box">
+          <div><img class="about-img" src="./Images/icon3.svg" alt="" /></div>
+          <div class="about-menu">
+            <h2 class="about-title">Skills</h2>
+            <ul class="about-list">
+              <li class="about-list-item">Codekit</li>
+              <li class="about-list-item">Github</li>
+              <li class="about-list-item">Codepen</li>
+              <li class="about-list-item">Gitlab</li>
+              <li class="about-list-item">Terminal</li>
+            </ul>
+          </div>
+        </li>
```

Comment:
As the last code reviewer requested you make your styling exactly in accordance with the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988&t=XlyksXSX1yjN1mJH-0) it's still not the same. I request you make them an exact copy of the design. To achieve this let me give you some specific guidance as you are very close to APPROVAL.

>1- Please change the background color of the `.about-box class` like this:
`.about-box {
    background: #EBF0EE;
}`

2- And for the `.about-list-item class` add the following:-
`.about-list-item {
    padding: 12px;
    border-radius: 8px;
}`

3- After that adjust the height and width of the container and make some `min-height` and `min-width` so it can increase its size for the increase in content and view-port width.
`.about-box {
   min-width: 327px;
   min-height: 352px;
}`